app/models/organization.rb

```diff
@@ -68,6 +68,9 @@ class Organization < ApplicationRecord
     has_many :purchases
     has_many :requests
     has_many :storage_locations
+    has_many :taggings
+    has_many :product_drive_tags, -> { merge(Tagging.by_type("ProductDrive")).distinct },
```

Comment:
So here this should be a simple `has_many` instead of `:through`

Reply:
Is the refactor from 413a6e07af1f789775ff19f504d360a3d566d880 what you meant?

Reply:
Nope - see above. I meant moving `organization_id` to `tags`.